app/Http/Controllers/TaskController.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Http\Controllers;
+use App\Task;
+use Illuminate\Http\Request;
+use App\Http\Requests;
+
+class TaskController extends Controller
+{
+	public function tasks() {
+    $tasks = Task::orderBy('created_at', 'asc')->get();
+
+    return view('tasks', [
+        'tasks' => $tasks
+    ]);
```

Comment:
Nghiên cứu https://laravel.com/docs/5.6/views#passing-data-to-views để xử lý nha e